dts/bindings/input/st,stmpe811.yaml

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) 2023 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: STMPE811 I2C touchscreen controller
+
+compatible: "st,stmpe811"
+
+include: i2c-device.yaml
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      Interrupt GPIO. Used by the controller to signal touch data is
+      available. Active low.
+
+  screen-width:
+    type: int
+    default: 0
+    description: |
+      Screen width for scaling the reported coordinates.
+      Default: raw touchscreen resolution.
+
+  screen-height:
+    type: int
+    default: 0
+    description: |
+      Screen height for scaling the reported coordinates.
+      Default: raw touchscreen resolution.
+
+  raw-x-min:
+    type: int
+    default: 0
+    description: |
+      Signed raw X axis start for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: no offset.
+
+  raw-y-min:
+    type: int
+    default: 0
+    description: |
+      Signed raw Y axis start for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: no offset.
+
+  raw-x-max:
+    type: int
+    default: 4096
+    description: |
+      Raw X axis end for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: raw touchscreen resolution.
+
+  raw-y-max:
+    type: int
+    default: 4096
+    description: |
+      Raw Y axis end for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: raw touchscreen resolution.
```

Comment:
I think this default could be better handled by a DT_PROP_OR(..., 4096).

Reply:
It is now changed to `DT_INST_PROP_OR`.